src/inspector/models/notation/footnotes/footnotesettingsmodel.cpp

```diff
@@ -0,0 +1,34 @@
+#include "footnotesettingsmodel.h"
+
+#include "translation.h"
+
+using namespace mu::inspector;
+
+FootnoteSettingsModel::FootnoteSettingsModel(QObject* parent, IElementRepositoryService* repository)
+    : InspectorModelWithVoiceAndPositionOptions(parent, repository)
+{
+    setModelType(InspectorModelType::TYPE_FOOTNOTE);
+    setTitle(muse::qtrc("inspector ", "Footnote"));
+    setIcon(muse::ui::IconCode::Code::EXPRESSION);
```

Comment:
I understand this is a placeholder, but it would be good to get a real icon here before merging this (task for the design team)